src/main/resources/db/migration/common/V1_381__amend_object_hidden_reason_values.sql

```diff
@@ -0,0 +1,22 @@
+-- Update existing rows in the object_hidden_reason table
```

Comment:
file will need renamed to 382 as someone else has merged another 381

Reply:
done